spec/support/shared_contexts/package_setup.rb

```diff
@@ -32,3 +32,73 @@
   end
   let(:package) { order.shipments.first.to_package }
 end
+
+shared_context 'Canada package setup' do
```

Comment:
Are these going to be used anywhere in the specs? I like the idea, but a bit hesitant to include here unless they are used.